FinancialTracker/Firebase.swift

```diff
@@ -23,54 +23,8 @@ enum DBCollectionKey: String {
     case users
 }
```

Comment:
If you do not need the `FirebaseHandler` anymore, might as well delete this class.

Reply:
Just needed to be sure everything was fine before deletion